graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -108,6 +110,9 @@ public void run() throws IOException {
         for (int i = 0; i < parallelQueues; i++) {
             final String queueName = String.format(queue, i);
             channel.queueDeclare(queueName, true, false, false, null);
+            if (exchangeBind) {
+                channel.queueBind(queueName, exchange, routingKey);
```

Comment:
If the input cannot be started due to a missing exchange, there is a warning in the web interface that is enough to debug the issue. It actually says that the exchange does not exist.